### DISCUSSION TITLE
Discord links are all expired.

### DISCUSSION THREAD

**Discussion by caelinsutch** [electric-sql/electric#646] [Bug reports]
Is the discord community still active? If so - the invitation links are all expired.

**Comment by thruflo** [comment]
Hi, yup, the Discord is very much still active. Can you confirm which link you’re using and from where?

The links on the website should be https://discord.electric-sql.com which works for me. Does that work for you?

**Reply by caelinsutch** [reply]
Weird - must have been my browser (Arc) or IP getting blocked. Tried it on an incognito tab and it worked.

**Comment by thruflo** [comment]
Odd, I wonder if the problem was with the redirect or Discord. Anyway, great it’s working now and thanks for flagging up.
